setup.cfg

```diff
@@ -0,0 +1,66 @@
+[metadata]
+name = ansible
+url = https://ansible.com/
+description = Radically simple IT automation
+long_description = file: README.rst
+author = Ansible, Inc.
+author_email = info@ansible.com
+license = GPLv3+
+classifiers =
+    Development Status :: 5 - Production/Stable
+    Environment :: Console
+    Intended Audience :: Developers
+    Intended Audience :: Information Technology
+    Intended Audience :: System Administrators
+    License :: OSI Approved :: GNU General Public License v3 or later (GPLv3+)
+    Natural Language :: English
+    Operating System :: POSIX
+    Programming Language :: Python :: 2.6
+    Programming Language :: Python :: 2.7
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Topic :: System :: Installation/Setup
+    Topic :: System :: Systems Administration
+    Topic :: Utilities
+keywords =
+    ansible
+    automation
+    infrastructure
+
+[options]
+python_requires = >=2.6,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*
```

Comment:
3.4 is also not supported.

Reply:
yea, missed that while focusing on other stuff :)